lib/ansible/module_utils/basic.py

```diff
@@ -379,9 +367,6 @@ def _load_params():
         print('\n{"msg": "Error: Module unable to decode stdin/parameters as valid JSON. Unable to parse what parameters were passed", "failed": true}')
         sys.exit(1)
 
-    if PY2:
-        params = json_dict_unicode_to_bytes(params)
```

Comment:
The unconditional import of `json_dict_unicode_to_bytes` can also be dropped (and turned into a deprecation).